docker-compose.prod.yml

```diff
@@ -1,17 +0,0 @@
-version: '3.8'
-
-services:
-  db:
-    restart: always
-    environment:
-      MYSQL_DATABASE: 'momentum'
-      MYSQL_USER: ${MOM_DATABASE_USER}
-      MYSQL_PASSWORD: ${MOM_DATABASE_PW}
-      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PW}
-
-  api:
-    image: docker.pkg.github.com/momentum-mod/website/mmod-website:staging
-    volumes:
-      - ./server/public/img/maps:/app/server/public/img/maps
-      - ./server/public/maps:/app/server/public/maps
-      - ./server/public/runs:/app/server/public/runs
```

Comment:
We don't need to delete this. I want to move production over to dislocated docker images like this, helps separate the responsibility and how much the VPS has to set up.

I'd re-add it but change the image: to be `ghcr.io/momentum-mod/website/mmod-website:staging` instead (using GH'd new URLs)

Reply:
Similarly, NODE_PORT needs defined, as production uses that in config.js